ospd_openvas/nvticache.py

```diff
@@ -143,7 +143,8 @@ def get_nvt_refs(oid):
 
 def get_nvt_prefs(ctx, oid):
     """ Get NVT preferences. """
-    prefs = ctx.smembers('oid:%s:prefs' % oid)
+    key = ('oid:%s:prefs' % oid)
+    prefs = ctx.lrange(key, 0, -1)
```

Comment:
what's the meaning of 0 and -1 values. Either use keyword arguments (aka. kwargs) or module constants for this.

Reply:
Maybe even use both kwargs and constants here.